### ISSUE TITLE
Delete plugin settings on uninstall

### ISSUE THREAD

**Issue by gruz0** [gruz0/widget-to-display-posts-from-current-category#10]

**Comment by rwkyyy** [comment]
if I have time, I'll try to make this. Although it should be noted via a settings panel. Sometimes you deactivate plugins for debugging purposes.

**Comment by gruz0** [comment]
Hi, Eduard!

We don't need to use settings page or something else, because WordPress has two different functions: `register_deactivation_hook` and `register_uninstall_hook` 😉 

If the user wants to deactivate plugin, then we will not delete the plugin's settings. Otherwise on uninstall we should delete our settings. What do u think about it?

Sure it is possible to create a settings page. But for what? 😄 

https://premium.wpmudev.org/blog/activate-deactivate-uninstall-hooks/

**Comment by rwkyyy** [comment]
Yes, I totally forgot about the uninstall hook, was thinking more at the deactivation hook.

Now the settings page (in this case), becomes redundant,
